spec/index.html

```diff
@@ -4614,8 +4614,7 @@ <h2>Expressions and Testing Values</h2>
 
 _:b   a:annotates   &lt;http://www.w3.org/TR/rdf-sparql-query/&gt; .
 _:b   dc:date       "2004-12-31T19:01:00-05:00"^^&lt;http://www.w3.org/2001/XMLSchema#dateTime&gt; .</pre>
-        <p>The object of the first <code>dc:date</code> triple is a 
-          <a data-cite="RDF12-CONCEPTS#dfn-simple-literal">simple literal</a>
+        <p>The object of the first <code>dc:date</code> triple is a literal
           that has a datatype of <code>xsd:string</code>.
           The second has the datatype <code>xsd:dateTime</code>.
           They are different <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF terms</a>
```

Comment:
```suggestion
          These are different <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF terms</a>
```

Reply:
Both are correct grammatically. "they" is simpler for ESL.

This is a case with "these" or "those" could be used, suggesting the neutral "they".